docker_mcp/core/transfer/archive.py

```
@@ -201,56 +201,66 @@ async def create_archive(
+                # Combine default and custom exclusions
+                all_exclusions = self.DEFAULT_EXCLUSIONS.copy()
+                if exclusions:
+                    all_exclusions.extend(exclusions)
+
+                # Build exclusion flags for tar
+                exclude_flags = []
+                for pattern in all_exclusions:
+                    exclude_flags.extend(["--exclude", pattern])
+
+                # Create timestamped archive name
+                timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
```

Ruff (0.14.4)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)